jdmkrt/core/src/main/java/com/sun/jdmk/ServiceName.java

```diff
@@ -178,22 +178,22 @@ private ServiceName() {
     public static final String JMX_IMPL_NAME = "Project OpenDMK";
 
     /**
-     * The name of the vendor of this product implementing the  JMX specification.  
+     * The name of the vendor of this product implementing the  JMX specification.
      * <BR>
      * The value is <CODE>Sun Microsystems</CODE>.
      */
     public static final String JMX_IMPL_VENDOR = "Sun Microsystems";
 
     /**
-      * The build number of the current product version, of the form 
+      * The build number of the current product version, of the form
       * <CODE>bXX</CODE>.
       */
-    public static final String BUILD_NUMBER = "b02";
```

Comment:
Not sure about this change. I do not recall how service lookups are made here, maybe that clashes with the binary plug? Maybe, for the entire project, it is better to increase the major (or just minor) version number to `2.0-b02` (or `1.1-b02`) from `1.0-b02` instead of the patch version number?

Reply:
It looks like it something used with JMX to identify the version of opendmk. I changed the BUILD_NUMBER back to "b02" and updated JMX_IMPL_VERSION.

```
 /**
     * The version of this product implementing the  JMX specification.
     * <BR>
     * The value is <CODE>opendmk-1.1-bXX</CODE>, where <CODE>bXX</CODE> is the
     * <CODE>BUILD_NUMBER</CODE> .
     */
    public static final String JMX_IMPL_VERSION = "opendmk-1.1-" + BUILD_NUMBER;
```